lobsterpy/cli.py

```diff
@@ -36,108 +37,159 @@ def get_parser() -> argparse.ArgumentParser:
         help='path to DOSCAR.lobster. Default is "DOSCAR.lobster".',
     )
 
-    input_file_group.add_argument(
-        "--vasprun",
+    user_basis_arg = argparse.ArgumentParser(add_help=False)
+    user_basis_arg.add_argument(
+        "--userbasis",
+        "--user-basis",
+        default=None,
+        type=_element_basis,
+        nargs="+",
+        help="Use the specific basis provided by the user to "
+        "generate the inputs (e.g.,  --userbasis Cr.3d.3p.4s N.2s.2p). "
+        "Default is None.",
+    )
+
+    # groups of arguments for specific actions
+
+    calc_quality_description_file_parent = argparse.ArgumentParser(add_help=False)
+
+    # Input args for specifically needed for calculation quality description
+    calc_quality_description_file_group = (
+        calc_quality_description_file_parent.add_argument_group()
+    )
+    calc_quality_description_file_group.add_argument(
+        "-fvasprun",
+        "--file-vasprun",
         default="vasprun.xml",
+        dest="vasprun",
         type=Path,
         help='path to vasprun.xml. Default is "vasprun.xml".',
     )
-
-    input_file_group.add_argument(
-        "--bandoverlaps",
+    calc_quality_description_file_group.add_argument(
+        "-fbandoverlap",
+        "--file-bandoverlaps",
         default="bandOverlaps.lobster",
+        dest="bandoverlaps",
         type=Path,
         help='path to bandOverlaps.lobster. Default is "bandOverlaps.lobster".',
     )
+    calc_quality_description_file_group.add_argument(
+        "-flobsin",
+        "--file-lobsterin",
+        default="lobsterin",
+        dest="lobsterin",
+        type=Path,
+        help='path to lobsterin. Default is "lobsterin".',
+    )
+    calc_quality_description_file_group.add_argument(
+        "-flobsout",
+        "--file-lobsterout",
+        default="lobsterout",
+        dest="lobsterout",
+        type=Path,
+        help='path to lobsterout. Default is "lobsterout".',
+    )
+    calc_quality_description_file_group.add_argument(
+        "-potsymbols",
+        "--potcar-symbols",
+        dest="potcarsymbols",
+        type=_potcar_symbols,
+        # nargs="+",
+        help="List of potcar symbols",
+    )
 
+    # group of arguments related to writing LOBSTER calcs inputs
     output_parent = argparse.ArgumentParser(add_help=False)
     output_file_group = output_parent.add_argument_group("Output files")
     output_file_group.add_argument(
-        "--incar-out",
-        "--incarout",
-        dest="incarout",
+        "-fincarout",
+        "--file-incar-out",
         default="INCAR.lobsterpy",
+        dest="incarout",
         type=Path,
-        help='path to INCAR that will be generated by lobsterpy. Default is "INCAR.lobsterpy"',
+        help='path to INCAR that lobsterpy generates. Default is "INCAR.lobsterpy"',
     )
     output_file_group.add_argument(
-        "--lobsterin-out",
-        "--lobsterinout",
-        dest="lobsterinout",
+        "-flobsinout",
```

Comment:
Can't it be -flobsterin as well? You will not read an old lobsterin at the same time, thus it can be the output as well

Reply:
Oh yes, makes sense . Will adapt it